### ISSUE TITLE
NullReferenceException in ComboBoxChildNativeWindow.WndProc

### ISSUE THREAD

**Issue by meyerrn** [microsoft/dotnet#1255]
In a WinForms project, we have a derived ComboBox which overrides CreateAccessibilityInstance(). After upgrading .NET Framework from 4.7.2 to 4.8, we face a unhandled NullReferenceException when mouse is moved over the opened combobox. This happens only if started from Visual Studio with debugger. The process is terminated, you cannot continue. In https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/ComboBox.cs,ffc826f5a6263a36 I can see that ComboBoxChildNativeWindow in WndProc assumes a specific type of AccessibilityObject, otherwise the cast will fail and NullReferenceException happen. I attached a minimal project to reproduce. Start from within Visual Studio. Can reproduce with both 2019 (16.7.5) and 2017 (15.9.27).
[CrashFramework4.8.zip](https://github.com/microsoft/dotnet/files/5355309/CrashFramework4.8.zip)
